src/main/java/org/folio/list/services/MigrationService.java

```diff
@@ -69,34 +70,27 @@ public boolean migrateList(ListEntity list) {
    * <strong>This does NOT check if this is actually necessary, and should only be invoked when the
    * caller is certain that lists need to be migrated.</strong>
    */
-  public CompletableFuture<Void> migrateAllLists() {
+  public List<CompletableFuture<Boolean>> migrateAllLists() {
     String tenant = executionContext.getTenantId();
 
-    return CompletableFuture.allOf(
```

Comment:
`CompletableFuture.allOf` discards all the original results?! Wild design choice, but that's why we're just slinging `List`s around here